Formula/consul-template.rb

```diff
@@ -16,50 +18,56 @@ class ConsulTemplate < Formula
 
   devel do
     url "https://github.com/hashicorp/consul-template.git",
-        :tag => "v0.18.0-rc1",
-        :revision => "933192dc0b6bb642af2549848e0681670550e095"
-    version "0.18.0-rc1"
-
-    # Upstream issue "Request: Makefile target that doesn't use docker"
-    # Reported 18 Nov 2016 https://github.com/hashicorp/consul-template/issues/793
-    patch :DATA
-  end
-
-  head do
-    url "https://github.com/hashicorp/consul-template.git"
-
-    patch :DATA
+        :tag => "v0.18.0-rc2",
+        :revision => "e446ab16fb567e1ddeb88ef29406b53bdb4ee387"
+    version "0.18.0-rc2"
   end
 
   depends_on "go" => :build
 
+  # This can be removed on next stable version
```

Comment:
These go_resource blocks should be moved into a `stable do` since they're only used by the stable spec. The block goes above the bottle block, and also should contain the stable url, :tag, :revision.